products/muleruntime.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Mule Runtime
+category: server-app
+iconSlug: mulesoft
+permalink: /muleruntime
+alternate_urls:
+  - /mulesoftruntime
+releasePolicyLink: https://www.mulesoft.com/legal/versioning-back-support-policy
+activeSupportColumn: Standard Support
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Extended Support
+changelogTemplate: https://docs.mulesoft.com/release-notes/mule-runtime/mule-__LATEST__-release-notes
+releaseLabel: "Mule Runtime (__RELEASE_CYCLE__)"
+releases:
+  - releaseCycle: "4.4"
+    eol: 2025-02-07
+    support: 2024-02-07
+    releaseDate: 2021-09-07
+    latest: "4.4.0"
+    latestReleaseDate: 2022-09-22
+
+  - releaseCycle: "4.3"
+    eol: 2025-03-07
+    support: 2023-03-07
+    releaseDate: 2020-04-30
+    latest: "4.3.0"
+    latestReleaseDate: 2022-09-22
+
+  - releaseCycle: "4.2"
+    eol: 2023-05-02
+    support: 2021-05-02
+    releaseDate: 2019-05-02
+    latest: "4.2.2"
+    latestReleaseDate: 2022-09-19
+
+  - releaseCycle: "4.1"
+    eol: 2022-11-02
+    support: 2020-11-02
+    releaseDate: 2018-03-20
+    latest: "4.1.6"
+    latestReleaseDate: 2021-04-19
+
+  - releaseCycle: "3.9"
+    eol: 2024-03-20
+    support: 2021-03-20
+    releaseDate: 2017-10-09
+    latest: "3.9.5"
+    latestReleaseDate: 2022-09-20
+    lts: true
+
+  - releaseCycle: "3.8"
+    eol: 2021-11-16
+    support: 2018-11-16
+    releaseDate: 2016-05-16
+    latest: "3.8.7"
+    latestReleaseDate: 2018-05-28
+    lts: true
+
+
+
+---
+
+> [MuleSoft Runtime](https://docs.mulesoft.com/mule-runtime/latest/) engine (Mule) is a lightweight integration engine that runs Mule applications and supports domains and policies. Mule applications, domains, and policies share an XML DSL (domain-specific language).
```

Comment:
It might be nice to have license information ( BSD ? MIT ? GPL ? Apache ? is this opensource eor not ? )
Also it will be much better to add What is LTS meaning for this product ? A few more info in a new paragraph about its release policy might be nice

Reply:
Looks proprietary so no mention is fine? I don't think we call out proprietary software anywhere.